python/setup.py

```diff
@@ -45,5 +45,7 @@ def get_lib_path():
       'xmltodict',
       'tabulate',
       'sodac',
+      'pandas',
+      'imageio',
```

Comment:
This package is only used in the test, right? If that's the case, you can modify [this file](https://github.com/cornell-zhang/heterocl/blob/master/.circleci/config.yml) instead. Just add a line `pip install --user imageio` before running the test.